packages/components/organisms/f-checkout/src/components/_tests/Error.test.js

```diff
@@ -42,4 +72,70 @@ describe('Error', () => {
         // Assert
         expect(description.text()).toMatchSnapshot();
     });
+
+    describe('methods', () => {
+        describe('`redirectToMenu` ::', () => {
+            const oldWindowLocation = window.location;
+
+            beforeAll(() => {
+                delete window.location;
+
+                window.location = Object.defineProperties(
+                    {},
+                    {
+                        ...Object.getOwnPropertyDescriptors(oldWindowLocation),
+                        assign: {
+                            configurable: true,
+                            value: jest.fn()
+                        }
+                    }
+                );
+            });
+
+            afterAll(() => {
+                window.location = oldWindowLocation;
+            });
+
+            it('should exist', () => {
+                expect(wrapper.vm.redirectToMenu).toBeDefined();
+            });
+
+            describe('when invoked', () => {
+                it('should assign the window location to the restaurant menu page url so the user can navigate back from the error', () => {
```

Comment:
Maybe `'should redirect the customer back to the menu'`?

Reply:
Done